### PR TITLE
MARKET-132 added demo

### PR DESCRIPTION
**Pull request by ivy-lmu**

### CODE REVIEW

File: docusign-connector-demo/processes/eSign.mod [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+st0 f25 responseCode 'import java.io.InputStream;
+InputStream inputStream = response.getEntity().toType(InputStream.class);
+
+String document = org.apache.commons.io.FilenameUtils.removeExtension(in.documents.get(0).name)+".pdf";
+ivy.case.documents().add(document).write().withContentFrom(inputStream);
+
+
+/*ivy.log.info("file: "+result);
+String document = org.apache.commons.io.FilenameUtils.removeExtension(in.documents.get(0).name)+".pdf";
+ivy.case.documents().add(document).write().withContentFrom(result);
```
Comment (ivy-lmu) [inline docusign-connector-demo/processes/eSign.mod:265]: @ivy-rew result is no more available. does it work with inputstream?

Comment (ivy-rew) [reply docusign-connector-demo/processes/eSign.mod:265]: works if you leave open api mode and set a type `java.io.File` in the result type

File: docusign-connector/src/com/axonivy/connector/docusign/DocUtils.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.runtime.CoreException;
+
+import com.docusign.esign.model.Document;
+
+import ch.ivyteam.ivy.application.IProcessModelVersion;
+import ch.ivyteam.ivy.scripting.objects.File;
+
+public class DocUtils
```
Comment (ivy-rew) [inline docusign-connector/src/com/axonivy/connector/docusign/DocUtils.java]: please format with our new coding style ( ivy-core/developement/eclipse-preferences.epf)

**Review by ivy-rew — APPROVED** [review]

**Comment by ivy-rew** [comment]
please remove the sample test: that the CI runs successfully.

**Pull request merged**